src/status_im/utils/js_require.clj

```diff
@@ -0,0 +1,9 @@
+(ns status-im.utils.js-require)
```

Comment:
is it a lazy-require macro? maybe name it that way to make it more explicit?

Reply:
kind of, ok